change-notes/1.23/analysis-cpp.md

```diff
@@ -2,44 +2,39 @@
 
 The following changes in version 1.23 affect C/C++ analysis in all applications.
 
-## General improvements
-
 ## New queries
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
-| Hard-coded Japanese era start date (`cpp/japanese-era/exact-era-date`) | reliability, japanese-era | This query is a combination of two old queries that were identical in purpose but separate as an implementation detail.  This new query replaces Hard-coded Japanese era start date in call (`cpp/japanese-era/constructor-or-method-with-exact-era-date`) and Hard-coded Japanese era start date in struct (`cpp/japanese-era/struct-with-exact-era-date`). |
-| Signed overflow check (`cpp/signed-overflow-check`) | correctness, security | Finds overflow checks that rely on signed integer addition to overflow, which has undefined behavior. Example: `a + b < a`. |
-| Pointer overflow check (`cpp/pointer-overflow-check`) | correctness, security | Finds overflow checks that rely on pointer addition to overflow, which has undefined behavior. Example: `ptr + a < ptr`. |
+| Hard-coded Japanese era start date (`cpp/japanese-era/exact-era-date`) | reliability, japanese-era | This query is a combination of two old queries that were identical in purpose but separate as an implementation detail.  This new query replaces Hard-coded Japanese era start date in call (`cpp/japanese-era/constructor-or-method-with-exact-era-date`) and Hard-coded Japanese era start date in `struct` (`cpp/japanese-era/struct-with-exact-era-date`). Results are not shown on LGTM by default. |
```

Comment:
I don't have strong feelings but I'm not convinced we should have backticks around "struct" in this case.  It _is_ a reference to a language construct, but it's also quoting the old name of the query (see https://github.com/Semmle/ql/pull/1847/files) and that didn't have backticks in it, so it seems like a misquote.

Reply:
The query is warning-medium, so it's run but not shown on LGTM by default.

Reply:
@jbj we should look into enabling this one actually; it rarely finds any results at all, but if we're running it anyway I think we might as well show the results.